schema/rdl_schema_0.1.json

```diff
@@ -579,6 +579,14 @@
 		"vulnerability_f_subtype": {
 			"type": "string",
 			"enum": ["Empirical", "Analytical", "Judgement", "Hybrid"]
+		},
+        "vulnerability_f_relationship": {
+			"type": "string",
+			"enum": ["TO Update"]
```

Comment:
Rather than leaving to-do's in the schema, it would be better to list these in an issue so that they are not missed (also applies to `#/$defs/vulnerability_f_math/enum`).

Reply:
And `#/$defs/common_exp_occupancy/enum`.